js/createPost.js

```diff
@@ -0,0 +1,4 @@
+import { PostCreator } from './classes/PostCreator.js';
+
+const createPost = new PostCreator(document.querySelector('.create-post-form'));
+console.log(createPost); // just for an avoiding linter error
```

Comment:
Don't understand why you need this `console.log`. Describe, please.

Reply:
I decided to use console.log to avoid ESLint error "value is assigned but never used"
![image](https://user-images.githubusercontent.com/47292994/91875249-ec723500-ec83-11ea-8162-bcc5b78804a2.png)